spec/attime_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'attime'
+
+describe '#attime' do
+  subject do |example|
+    delta = example.metadata[:description_args][0]
+    CheckGraphite.attime(delta)
+  end
+
+  def timedelta(n)
+    Time.now + n
```

Comment:
It hardly takes 0.9 secs to GC a test execution? But you are right, it makes for better specs. In fact, attime() should probably require the time since that is how it is used in production code.